contracts/interfaces/IDCAStrategies.sol

```diff
@@ -0,0 +1,249 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC721/extensions/IERC721Enumerable.sol';
+import '@mean-finance/nft-descriptors/solidity/interfaces/IDCAHubPositionDescriptor.sol';
+
+interface IDCAStrategiesBase {
```

Comment:
We normally have all shared structs/enum in the main interface (in this case `IDCAStrategies`), but I'd like to hear your thoughts. Do you think it would be better to have it like this?

Reply:
If I move shared structs then the handlers cannot find them. Do you have an example of how I need to do it?

Reply:
You would need to write `IDCAStrategies.{The other part}`

Like here: https://github.com/Mean-Finance/dca-v2-core/blob/main/contracts/DCAHub/DCAHubPositionHandler.sol#L62

Reply:
I didn't know that. Done!